### ISSUE TITLE
Description on form radio button group not accessible to screen reader

### ISSUE THREAD

**Issue by ccsaposs** [bootstrap-vue/bootstrap-vue#6095]
### Describe the bug

The description for a b-form-group containing a b-form-radio-group is not accessible to the ChromeVox screen reader.

### Steps to reproduce the bug

1. Activate ChromeVox (Ctrl+Alt+Z)
2. Go to https://jsfiddle.net/qw394szo/1/
3. Use the tab key to select the radio button group
4. ChromeVox reads "option a, radio button unselected; 1 of 2; Radio button group; Label; Form;"

### Expected behavior

ChromeVox should have read "option a, radio button unselected; 1 of 2; Radio button group; Label; Form; **Description**"

### Versions

**Libraries:**

- BootstrapVue: 2.19.0
- Bootstrap: 4.5.3
- Vue: 2.6.12

**Environment:**

- Device: Chromebook
- OS: ChromeOS
- Browser: Chrome
- Version: 86

### Demo link

https://jsfiddle.net/qw394szo/1/

### Additional context

Similar to https://github.com/bootstrap-vue/bootstrap-vue/issues/5918#issuecomment-734434869
